tests/nested/manual/core20-remodel/task.yaml

```diff
@@ -0,0 +1,116 @@
+summary: verify a simple UC20 remodel
+
+systems: [ubuntu-20.04-64]
+
+environment:
+  NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/valid-for-testing-pc-20.model
+  NESTED_IMAGE_ID: uc20-remodel-testing
+  NESTED_ENABLE_TPM: true
+  NESTED_ENABLE_SECURE_BOOT: true
+  NESTED_BUILD_SNAPD_FROM_CURRENT: true
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    # modify and repack gadget snap to add a defaults section and use our own
+    # prepare-device hook to use the fakedevicesvc
```

Comment:
this is not the case here :smile:

Reply:
also afaik, nested.sh when building the image will already sign the gadget snap properly, so we don't need any of this and this can just be `tests.nested build-image core` and `tests.nested create-vm core`, no?

Reply:
Yeah, good catch. Let me drop that and see whether the test is passing still.